docs/setup/native.md

```diff
@@ -88,36 +94,43 @@ This file has the necessary configuration settings for local development as well
 
 ### Configuring ClamAV antivirus
 
-In many cases, there in no need to run ClamAV for local development, even if you are working with uploaded files since the scanning functionality is already built into our CarrierWave and Shrine file upload base classes.
```

Comment:
I liked the previous mention of, "Hey you probably don't need to do this locally". Thoughts?